bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooabspdf.py

```diff
@@ -10,45 +10,34 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-r"""
```

Comment:
Ok so the idea is to remove all the doxygen documentation specified as a string at the beginning of the module so that the current PyROOT doc generation mechanism does not pick it anymore, and you will pick it from the class and its methods. Is this correct?

Reply:
Exactly!